starboard/android/shared/video_window.cc

```diff
@@ -229,7 +238,6 @@ void VideoSurfaceHolder::ClearVideoWindow(bool force_reset_surface) {
       return;
     }
   }
-  ClearNativeWindow(g_native_video_window);
```

Comment:
Just to double check:
1. Shall we still call `resetVideoSurface()` above?
2. Instead of not calling `ClearNativeWindow()` inside this function, shall we just avoid calling `ClearVideoWindow()` from the `VideoDecoder` dtor?

My main concern here is, while the current approach may work, it's slightly weird for the `VideoDecoder` to call a function named `ClearVideoWindow()`, but the function actually doesn't clear the video surface.

Also, we've been using this in Cobalt for years, shall we just resolve the 20 seconds suspend/resume issue or is that intended?

Reply:
Setting `overlay_info_.is_fullscreen` is a clean fix to this issue.